README.md

```diff
@@ -1,8 +1,8 @@
-# tap-facebook
+# `tap-facebook`
 
-`tap-facebook` is a Singer tap for facebook.
+facebook tap class.
```

Comment:
This doesn't seem right 😅. You may want to update the docstring of `Tapfacebook` (and the class name while we're at it).